pom.xml

```diff
@@ -1131,7 +1131,7 @@
           <configuration>
             <scalaVersion>${scala.version}</scalaVersion>
             <recompileMode>incremental</recompileMode>
-            <useZincServer>true</useZincServer>
+            <useZincServer>false</useZincServer>
```

Comment:
Sorry i made a mistake. I'll revert it.

2015-03-25 19:05 GMT+08:00 Sean Owen <notifications@github.com>:

> In pom.xml
> <https://github.com/apache/spark/pull/5185#discussion_r27111088>:
>
> > @@ -1131,7 +1131,7 @@
> >            <configuration>
> >              <scalaVersion>${scala.version}</scalaVersion>
> >              <recompileMode>incremental</recompileMode>
> > -            <useZincServer>true</useZincServer>
> > +            <useZincServer>false</useZincServer>
>
> without commenting on the rest, this should be reverted
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/5185/files#r27111088>.
>